guide/payments/units-and-symbols.md

```diff
@@ -0,0 +1,198 @@
+| ------------ | ------------ | ----------------- |
+| bitcoin      | BTC or ₿     | 1                 |
+| millibit     | mBTC         | 0.001             |
+| bit          | μBTC         | 0.000 001         |
+| satoshi      | sat          | 0.000 000 01      |
+| millisatoshi | msat         | 0.000 000 000 01  |
+
+For more information, see the Bitcoin Wiki:
+
+- [Units](https://en.bitcoin.it/wiki/Units)
+- [Satoshi](https://en.bitcoin.it/wiki/Satoshi_(unit))
+
+## Recommended interaction
+
+When displaying bitcoin values, the default unit for on-chain wallets should be bitcoin with 8 decimal places, and satosho for Lightning wallets. Due to the challenging nature of scanning a monetary value with more than 2 decimal places, the user should be given the option to choose their preferred format across the application (for example, in the settings section) as well as contextually, whenever the value is primarily displayed.
+
+Product teams can choose an approach based on their audience and targeted use case. Lightning wallets for daily spending may be better served by defaulting to satoshi due to the low amounts involved, while bitcoin can be used for savings-focused applications.
+
+### Setting the preferred unit
+
+Applications should try to establish smart defaults that work for the majority of users, while offering convenient ways to customize settings.
+
+{% include image-gallery.html pages = page.images_app %}
+
+### Changing the unit display contextually
+
+User should always be able to change unit contextually. Additionally, for the value to have meaning to most users today, the option to select the local currency should also be readily available.
+
+{% include image-gallery.html pages = page.images_home %}
+
+## Formatting units
+
+{% include picture.html
+   image = "/assets/images/guide/payments/units-and-symbols/formatting-elements.png"
+   retina = "/assets/images/guide/payments/units-and-symbols/formatting-elements@2x.png"
+   alt-text = "Bitcoin island travel map"
+   width = 800
+   height = 300
+%}
+
+There are many different ways of formatting numbers and currency units across the world. Bitcoin applications should be sensitive to these standards by adapting the formatting to the user's locale. A simple example are digit group and decimal separators. Based on where you live you may be accustomed to one of these formatting standards, but be less comfortable with the others:
+- `10 000,00`
+- `10.000,00`
+- `10,000.00`
+- `10'000,00`
+- `10,000·00`
+
+For more examples and information, see Wikipedia:
+
+- [Currency symbol](https://en.wikipedia.org/wiki/Currency_symbol)
+- [Unit prefix](https://en.wikipedia.org/wiki/Unit_prefix)
+- [Decimal separator](https://en.wikipedia.org/wiki/Decimal_separator)
+
+## Visual styling
+
+Type choices, spacing, color, and other details also affect the ease at which users can understand written amounts. Below are some options to consider.
+
+### Digit groups
+
+{% include picture.html
```

Comment:
The contrast in the alternate coloring isn't clear, I couldn't see the difference at a first glance.

Reply:
Yes, I went back and forth on that. Too strong of a contrast results makes the numbers hard to read, and too little contrast is barely noticeable. I went for low contrast after experimenting a bit, as it's meant to be a subtle helper and work almost subconsciously. I think designers need to adjust this based on the foreground/background colors they use, as well as font details (thin vs. bold).